bindings/python/pmix.pyx

```diff
@@ -251,6 +251,69 @@ cdef class PMIxClient:
             pmix_free_info(info, ninfo)
         return rc
 
+    # put a value into the keystore
+    #
+    # @proc [INPUT]
+    #       - namespace and rank of the client (tuple)
+    #
+    # @ninfo [INPUT]
+    #      - number of entries in dicts list
+    #
+    # @key [INPUT]
+    #      - the key to be stored
+    #
+    # @dicts [INPUT]
+    #          - a list of dictionaries, where each
+    #            dictionary has a key, value, and val_type
+    #            defined as such:
+    #            [{key:y, value:val, val_type:ty}, … ]
+    #
+    # @value [INPUT]
+    #        - a value tuple to be stored (value, type)
+    def get(self, proc:tuple, ky, dicts:list, num_info, val:tuple):
```

Comment:
There should be no "val:tuple" in this signature. We will take the inputs and return a tuple containing the value returned from the keystore